src/App.js

```diff
@@ -291,6 +304,10 @@ class App extends Component {
   toMainView = () => {
     this.setState({ 'currentView': View.DASHBOARD });
   }
+  
+  getLayoutsList = () => {
+    return (Object.keys(this.store.ls).filter(function (propertyName) { return propertyName.indexOf("layout") === 0;}));
```

Comment:
Can you please move Object.keys(this.store.ls) in a new getKeys function of the localstorage helper class?